pages/Threat_Modeling.md

```diff
@@ -142,18 +104,16 @@ argument that can be used to explain and defend the security of an
 application. An assurance argument starts with a few high level claims,
 and justifies them with either subclaims or evidence.
 
-## ThreatModel SDK
+## ThreatModel Tooling
 
-The ThreatModel SDK (<https://github.com/stevespringett/threatmodel-sdk>)
-is a minimalistic Java library that provides a basic
+There is a wide variety of tooling for threat modeling, including OWASP Threat Dragon and "The ThreatModel SDK" (<https://github.com/stevespringett/threatmodel-sdk>)
+which is a minimalistic Java library that provides a basic
 vendor-neutral object model along with the ability to parse reports
 generated from common threat modeling tools. It currently supports:
```

Comment:
Should probably add https://owasp.org/www-project-threat-dragon/

Reply:
Added link; I don't think I've added enough to list myself as a contributor so will pass on that.